android/src/main/java/com/facebook/flipper/plugins/sharedpreferences/SharedPreferencesFlipperPlugin.java

```diff
@@ -128,10 +130,18 @@ public boolean accept(File dir, String name) {
         String prefName = each.substring(0, each.indexOf(XML_SUFFIX));
         descriptors.add(new SharedPreferencesDescriptor(prefName, MODE_PRIVATE));
       }
-    }
+    }    
+
+    descriptors.add(new SharedPreferencesDescriptor(getDefaultSharedPreferencesName(context), MODE_PRIVATE));
+          
     return descriptors;
   }
 
+  private static String getDefaultSharedPreferencesName(Context context) {
+      return Build.VERSION.SDK_INT >= Build.VERSION_CODES.N
+              ? PreferenceManager.getDefaultSharedPreferencesName(context)
+              : context.getPackageName() + "_preferences";
+  
```

Comment:
I think you're missing a curly brace here to end the method, which is why the CI builds are failing.